content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -12,8 +12,7 @@ card:
 
 ## {{% heading "prerequisites" %}}
 
-You must use a kubectl version that is within one minor version difference of your cluster.
-For example, a v1.2 client should work with v1.1, v1.2, and v1.3 master.
+You must use a kubectl version that is within one minor version difference of your cluster. For example, a v{{< skew latestVersion >}} client can communicate with v{{< skew latestVersion >}}, v{{< skew prevMinorVersion >}} and v{{< skew nextMinorVersion >}} control planes.
```

Comment:
Hi @Kartik494 . Thanks for contributing.
Looking at the preview pages, what do you think about listing the versions in order,
v{{< skew prevMinorVersion >}}, v{{< skew latestVersion >}}, and v{{< skew nextMinorVersion >}}?

Reply:
@kbhawkey Thanks for the input , i will update it accordingly

Reply:
@kbhawkey   i have addressed the changes . PTAL